web-api/README.md

```diff
@@ -148,6 +148,28 @@ console.log(response.accessToken);
 
 or using some oauth2 library like [client-oauth2](https://www.npmjs.com/package/client-oauth2)
 
+### Accessing API endpoints as a Calendar App
+
+```ts
+import { OAuthAuthenticator, CalendarAppClient } from "@timetreeapp/web-api";
+
+// Generate and download the private key. Please refer to the link below.
+// https://developers.timetreeapp.com/docs/api/calendar-app
+const authenticator = new CalendarAppAuthenticator({
+  applicationId: "<your-jwt-application-id>",
```

Comment:
The documentation seems to name it `CALENDAR_APP_ID`
https://developers.timetreeapp.com/en/docs/api/calendar-app#when-a-user-unlinks-your-calendar-app

```suggestion
  applicationId: "<your-calendar-app-id>",
```